tools/editor/plugins/shader_graph_editor_plugin.cpp

```diff
@@ -1382,7 +1386,8 @@ ToolButton *ShaderGraphView::make_editor(String text,GraphNode* gn,int p_id,int
 		t.instance();
 		t->create_from_image(icon_color);
 		edit->set_icon(t);
-		break;
+	} break;
```

Comment:
Does that change the logic? If not it's probably not necessary, as the previous format was more consistent with what's used above?